test/athens/cljs_parser_test.cljs

```diff
@@ -1,816 +0,0 @@
-(ns athens.cljs-parser-test
```

Comment:
Why do we want to get rid of it?

Reply:
I rolled it into the `test/athens/parser/impl_test.clj -> impl_test.cljc` change, but am not sure if that's the right thing to do. Seemed to be the same tests except for the clj/cljs differences.